test/box/gh-6290-bg-alter-pk-mem-leak.test.lua

```diff
@@ -0,0 +1,61 @@
+env = require('test_run')
+test_run = env.new()
+test_run:cmd("create server bg_index_mem_leak with script='box/gh-6290-bg-alter-pk-mem-leak.lua'")
+test_run:cmd("start server bg_index_mem_leak")
+test_run:cmd("switch bg_index_mem_leak")
+
+fiber = require('fiber')
+
+s = box.schema.space.create('test', {engine = 'memtx'})
+_ = s:create_index('pk',  {type='tree', parts={{1, 'uint'}}})
+
+h = box.schema.space.create('heavy', {engine = 'memtx'})
+_ = h:create_index('pk')
+
+for i = 1, 150000 do h:replace{i, "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa 100 bytes"} end
```

Comment:
Still test runs a bit for a long time: on my i7-8700 it takes tens seconds. Can you short memtx_memory a few times more?

Reply:
Ok than I'd mark this test as long-run..